galleries/examples/images_contours_and_fields/image_exact_placement.py

```diff
@@ -0,0 +1,167 @@
+"""
+=========================================
+Placing images, preserving relative sizes
+=========================================
+
+By default Matplotlib resamples images created with `~.Axes.imshow` to
+fit inside the parent `~.axes.Axes`.  This can mean that images that have very
+different original sizes can end up appearing similar in size.
```

Comment:
Optional: Slightly more concise wording:
```suggestion
fit inside the parent `~.axes.Axes`.  As a result, images with very different
original sizes may end up appearing similar in size.
```